README.md

```diff
@@ -180,17 +180,29 @@ depending on whether you want to use HTTPS or SSH. Finally, run the
 following:
 
 ```sh
-
 python3 -m pip install -r requirements.txt
 ```
 
 ## Usage
 
 To run your first example Scene, you can run the following commands:
-For users:
-1. Download the `example_scenes.py` file from [GitHub](https://github.com/ManimCommunity/manim/blob/master/example_scenes.py)
+
+### For users:
+
+1. Download the `example_scenes/basic.py` file from [GitHub](https://github.com/ManimCommunity/manim/blob/master/example_scenes/basic.py)
 2. Run the following command:
 ```sh
+manim example_scenes/basic.py SquareToCircle -pl
+```
+
+### For developers:
+
+- Run the following command to install manim:
+```sh
+pip install -e .
+```
+- Then run the following command below:
+```sh
 manim example_scenes.py SquareToCircle -pl
```

Comment:
It's `manim example_scenes/basic.py SquareToCircle -pl`

Reply:
@kilacoda add the suggestion thing

Reply:
Uhh... how?

Reply:
Oh ok nvm found it

Reply:
Done